akka-stream-tests/src/test/scala/akka/stream/impl/PartitionedBufferSpec.scala

```diff
@@ -9,13 +9,12 @@ import akka.stream.testkit.StreamSpec
 class PartitionedBufferSpec extends StreamSpec {
   val doNothing: Any => Unit = _ => ()
 
-  val linearCapacity = scala.util.Random.nextInt(32) + 1
+  /*
```

Comment:
?

Reply:
Ups, left the code for reference while simplifiying. Intended to remove.